functions/firebase/Firebase.Tests/Firebase.Tests.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Google.Cloud.Functions.Invoker.Testing" Version="1.0.0-alpha11" />
```

Comment:
You are missing the JUnit dependency, that's why the Firebase tests are failing.